projects/angular/src/layout/grid/STYLES.md

```diff
@@ -1,9 +1,19 @@
 # Grid layout
 
+## CSS Custom Properties
+
+| CSS Custom Property          | Description                                                              |
+| ---------------------------- | ------------------------------------------------------------------------ |
+| --clr-grid-gutter-width      | Grid gutter width                                                        |
+| --clr-grid-container-padding | Grid container padding size. ( Default value is --clr-grid-gutter-width) |
+| --clr-grid-column-padding    | Grid column padding size. ( Default value is --clr-grid-gutter-width)    |
+| --clr-grid-row-margin        | Grid row margin size. ( Default value is --clr-grid-gutter-width)        |
```

Comment:
In the `projects/angular/src/utils/variables/_properties.layout.scss` file the default values are actually half of the `--clr-grid-gutter-width`.


```suggestion
| --clr-grid-container-padding | Grid container padding size. ( Default value is half of --clr-grid-gutter-width) |
| --clr-grid-column-padding    | Grid column padding size. ( Default value is half of --clr-grid-gutter-width)    |
| --clr-grid-row-margin        | Grid row margin size. ( Default value is half of --clr-grid-gutter-width)        |
```

Reply:
And `--clr-grid-row-margin` is negative.

Reply:
Thanks for catching it! The values were different and I forgot to change the descriptions after I changed them.

Reply:
Changed in [7079953](https://github.com/vmware-clarity/ng-clarity/pull/1457/commits/7079953691033cf5a38266ce82c7f51be0f19485)